engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -132,6 +133,16 @@ private ConfigurationImpl() {
 		this.addConstraintDefinitionContributor( defaultConstraintDefinitionContributor );
 	}
 
+	private JavaFXPropertyValueUnwrapper createJavaFXUnwrapperClass(TypeResolutionHelper typeResolutionHelper, ClassLoader classLoader) {
+		try {
+			Class<?> jfxUnwrapperClass = classLoader.loadClass( "org.hibernate.validator.internal.engine.valuehandling.JavaFXPropertyValueUnwrapper" );
```

Comment:
All classloading has to be done via a `PrivilegedAction`. We cannot call `ClassLoader.loadClass` directly.